lib/checks/shared/filename-is-valid-accessible-name.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "filename-is-valid-accessible-name",
+	"evaluate": "filename-is-valid-accessible-name.js",
+	"metadata": {
+		"impact": "minor",
+		"messages": {
+			"pass": "Element has an accessible name that serves an equivalent purpose to the non-text content",
+			"incomplete": "Element does not have an accessible name that serves an equivalent purpose to the non-text content"
```

Comment:
This doesn't help me understand why the check "failed" if the image uses the file extension in the acc name. Should tell me how to fix it so I don't have to click out to the linked help doc.

Reply:
@jeankaplansky - I need help with better messages here.
Here is more info about the rule here - https://act-rules.github.io/rules/9eb3f6